File: JS/Queue_Stack/Stack.js
class Stack
{

    constructor()
    {
        this.items = [];
    }
    push(element)
    {
        this.items.push(element);
    }
    pop()
    {
        return this.items.pop();
    }
    peek()
    {
        return this.items[this.items.length - 1];
    }
    length()
    {
        return this.items.length ;
    }

}


s = new Stack();
s.push(5);
s.push(51);
s.push(55);
s.push(59);

for(i=0;i<4;i++)
{
    console.log(s.pop());
}

File: Java/Queue_Stack/Stack.java
package Java.Queue_Stack;

import java.util.ArrayList;

public class Stack {
     public Stack()
    {
        m_arr = new ArrayList<Integer>();
    };

    public void Push(int item)
    {
        m_arr.add(item);
    }

    public int Pop()
    {
        int n = m_arr.get(Size()-1);
        m_arr.remove(Size()-1);
        return n;
    }

    public int Size()
    {
        return m_arr.size();
    }


    private ArrayList<Integer> m_arr;
    
}

